app/views/layouts/_drawer.mobile.haml

```diff
@@ -35,5 +35,22 @@
           = person_image_tag(current_user, size: :thumb_small)
       %li= link_to t("_contacts"), contacts_path
       %li= link_to t("layouts.header.settings"), edit_user_path
+      - if current_user.admin?
+        %li#admin
+          = link_to t("streams.admin.title"), "#"
+        %li.no-border.hide
+          %ul
+            %li= link_to t("admins.admin_bar.dashboard"), admin_dashboard_path
+            %li= link_to t("admins.admin_bar.user_search"), user_search_path
+            %li= link_to t("admins.admin_bar.weekly_user_stats"), weekly_user_stats_path
+            %li= link_to t("admins.admin_bar.pod_stats"), pod_stats_path
+            %li{role: "presentation", class: current_page?(report_index_path)}
```

Comment:
It's a copy/paste from [desktop view's menu](https://github.com/diaspora/diaspora/blob/795283b581d6e9d2ec93f150a00a8efe90779814/app/views/admins/_admin_bar.haml#L6). My ruby/HAML skills are **really** limited, so I try many things, and when it works, I don't touch anymore.

BTW, thank you very much for your review/time @cmrd-senya!